### DISCUSSION TITLE
LAMMPS installation

### DISCUSSION THREAD

**Discussion by nema9210** [Homebrew/discussions#3278] [Getting started] [answered]
Hello All,

I just installed Homebrew on my macOS to install LAMMPS software and I did installed LAMMPS successfully.I followed the instruction on following link:

https://docs.lammps.org/Install_mac.html

My question is how do I make sure that a special package I need for example in this case {DPD-REACT} is installed and if it is not how do I do it, using homebrew commands?

Thank you

**Comment by SMillerDev** [comment, marked as answer]
`brew install <name>` installs a package with the given name.

**Reply by nema9210** [reply]
Thank you so much, but I got the following Errors 

Warning: No available formula with the name "dpd-react".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

**Reply by nema9210** [reply]
I wrote 

brew install DPD-REACT

So does it figure out it needs to be a lammps package?
Is that the reason I am getting the following errors?

**Reply by SMillerDev** [reply]
If there is a homebrew package like that it'll install it. If it doesn't exist it won't. If lammps has a specific way to install packages that's probably in the lammps documentation
